code/modules/projectiles/projectile/bullets.dm

```diff
@@ -115,16 +115,17 @@
 	weaken = 0
 
 /obj/item/projectile/bullet/fourtyfive //secgun ammo!
-	damage = 15
-	stun = 2 //stopping power
-	weaken = 2
+	damage = 13
+	drowsy = 0.25 //stopping power
```

Comment:
0.25 seems incredibly tiny, drowsyness is reduced by 1 every life tick. And agony on projectiles deals hallucination damage, 0.25 of that might as well not exist